rholang/src/main/k/rholang/name-equivalence.k

```diff
@@ -0,0 +1,13 @@
+module NAME-EQUIVALENCE
+imports FREE
+
+// A function that determines if two names are equivalent
+// This will be used to determine equivalence on a send/receive
+// Names are equivalent, modulo:
```

Comment:
Is `x!(0)` equivalent to `*@{x!(0)}`?

Reply:
In what context, as processes? e.g., are `@{ x!(0) }` and `@{ *@{ x!(0) }}` equivalent channels to send/receive on? I presume so. This module is meant to define a function that looks at names being sent/received on to see if they are equivalent.

Reply:
Certainly if `x` is bound, a send on the first will match with a receive on the second, so long as the pattern matches the process being sent.

Reply:
Ah, I see your point. I will add that to the list.